src/com/hapgood/refreshable.cljc

```diff
@@ -130,6 +184,12 @@
 
 (def close! async/close!)
 
+(def closed? impl/closed?)
```

Comment:
This was not exposed because core.async does not expose such a function.  I'm not sure that it is safe.  If you only need it for testing, perhaps write a test helper to peer into the guts of the refreshable?